src/Components/FeatherIcons.re

```diff
@@ -0,0 +1,148 @@
+open Revery;
+open Revery.UI.Components;
+
+let colorToCSS = color => {
+  let (r, g, b, _) = Color.toRgba(color);
+  Printf.sprintf(
+    "#%02X%02X%02X",
+    int_of_float(r *. 255.),
+    int_of_float(g *. 255.),
+    int_of_float(b *. 255.),
+  );
+};
+
+let folder: SVGIcon.t =
+  (~size=24, ~strokeWidth=2, ~color=Colors.white, ()) =>
+    <SVG
+      src={
+            `Str(
+              Printf.sprintf(
+                {|<svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  width="%d"
+                  height="%d"
+                  viewBox="0 0 %d %d"
+                  fill="none"
+                  stroke="%s"
+                  stroke-width="%d"
+                  stroke-linecap="round"
+                  stroke-linejoin="round"
+                  class="feather feather-folder"
+                >
+                  <path d="M22 19a2 2 0 0 1-2 2H4a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h5l2 3h9a2 2 0 0 1 2 2z" />
+                </svg>
+            |},
+                size,
+                size,
+                size,
+                size,
+                colorToCSS(color),
+                strokeWidth,
+              ),
+            )
+          }
+    />;
+
+let codeSandbox: SVGIcon.t =
```

Comment:
We'll need to include the license in the `ThirdPartyLicenses.txt` - https://github.com/feathericons/feather/blob/master/LICENSE

(We should put a link the source file here too)

Reply:
Thanks!